modules/client/src/addresslistVoting/internal/constants.ts

```diff
@@ -1,19 +1,12 @@
 import {
   AddresslistVoting__factory,
-  MajorityVotingBase__factory
+  MajorityVotingBase__factory,
 } from "@aragon/core-contracts-ethers";
 
-// NOTE: This address needs to be set when the plugin has been published and the ID is known
-// TODO: Find a way around this
-// This shouldnt be hardcoded, but the active contracts need the current network
-// thes means we cannot have it here if we move it to the function where this is used we 
-// still dont have access to the network because is a static function and the context is 
-// not defined. Maybe defining the ens here instead of the address?
-export const ADDRESSLIST_PLUGIN_ID =
```

Comment:
Why not having the constants here, per-plugin? They will be split into separate packages at some point

Reply:
Right now we have it in the common client package, we can split it if you want

Reply:
Let's create a follow-up task then. Please resolve